tests/macsec/macsec_config_helper.py

```diff
@@ -76,22 +76,33 @@ def enable_macsec_port(host, port, profile_name):
             lines=['mac security profile {}'.format(profile_name)],
             parents=['interface {}'.format(port)])
         return
+    
+    pc = find_portchannel_from_member(port, get_portchannel(host))
+    if pc:
+        host.command("sudo config portchannel {} member del {} {}".format(getns_prefix(host, port), pc["name"], port))
 
-    cmd = "sonic-db-cli {} CONFIG_DB HSET 'PORT|{}' 'macsec' '{}'".format(
-        getns_prefix(host, port), port, profile_name)
+    cmd = "sonic-db-cli {} CONFIG_DB HSET 'PORT|{}' 'macsec' '{}'".format(getns_prefix(host, port), port, profile_name)
     host.command(cmd)
 
-
+    if pc:
+        host.command("sudo config portchannel {} member add {} {}".format(getns_prefix(host, port), pc["name"], port))
+        
 def disable_macsec_port(host, port):
     if isinstance(host, EosHost):
         host.eos_config(
             lines=['no mac security profile'],
             parents=['interface {}'.format(port)])
         return
 
+    pc = find_portchannel_from_member(port, get_portchannel(host))
+    if pc:
+        host.command("sudo config portchannel {} member del {} {}".format(getns_prefix(host, port), pc["name"], port))
+        
```

Comment:
Should we wait few seconds with same reason as above?

Reply:
done